src/main/java/lesson1/Homework1/Task2_dz1.java

```diff
@@ -0,0 +1,17 @@
+package lesson1.Homework1;
+
+import java.util.Scanner;
+
+public class Task2_dz1 {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Радиус круга(м):");
+        float r = sc.nextInt();
+        double S;
+        double pi;
+        pi = 3.14;
+        S = Math.pow(r, 2) * pi;
```

Comment:
you can write it in 1 line 
 `double S = Math.pow(r, 2) * pi;`

also in class Math you can find some constants, such as Math.PI and use it, e.g.
 `double S = Math.pow(r, 2) * Math.PI;`